### DISCUSSION TITLE
TACOS "EL JEFE"

### DISCUSSION THREAD

**Discussion by floressdi** [Launch-X-Latam/MisionFrontEnd#459] [Team Posts]
¡Hola Gente!
les presento la primera version sobre Tacos EL JEFE
https://floressdi.github.io/Tacos-EL-JEFE-/

Utilize un poco de trampa con Tailwind css, Codigo Javascript  y una plantilla que ya tenia de un proyecto por si quieren hecharle un ojo acepto feedback.

**Comment by AlexLopez24** [comment]
Esta genial :3 el logo esta bien perro xd 
solo una duda esa que practica es? yo me he estado guiando por el repositorio pero no veo nada de tacos el jefe :( sorry por no saber :"v

**Comment by floressdi** [comment]
¡Hola! Tacos EL JEFE es la practica 2 yo la vi en la sesion del 14 octubre pero apenas termine de ver el  live ayer por eso apenas termine la primera versión saludos.

**Comment by AlexLopez24** [comment]
Ohh entiendo, entonces lo de la pasteleria ahora es tacos el jefe! 
muy agradecido hermano por la info! :3

**Comment by brayandoria98** [comment]
No, son dos prácticas. Yo le pregunté al MS por Teams y el me respondió, la que él hizo en el live es una y la pastelería es otra. La de Tacos el Jefe no es necesario que trate de tacos, puede ser cualquier comida callejera.

**Comment by floressdi** [comment]
Entonces son 2 en total? 
Una de comida y otra de pasteleria?

**Comment by JoseEduardoVelazquezCN** [comment]
Pues oficialmente las prácticas son las que están en el repositorio no ¿? Ya va de ti como es autodidacta la experiencia pues al hacer las prácticas de los lives pues más experiencia vas formando y más proyectos para tu portafolio

**Comment by floressdi** [comment]
> Pues oficialmente las prácticas son las que están en el repositorio no ¿? Ya va de ti como es autodidacta la experiencia pues al hacer las prácticas de los lives pues más experiencia vas formando y más proyectos para tu portafolio

Ah ok es que no lo habia checado gracias!!

**Comment by Ghost3119** [comment]
Oye hermano donde aprendiste Tailwind css?. te quedo muyyyy bien. saludos

**Comment by KattGG** [comment]
Me encantó! Felicidades compañero.

**Comment by brayandoria98** [comment]
> Entonces son 2 en total?
> Una de comida y otra de pasteleria?

Sí, son dos. Yo le pregunté por interno.

**Comment by terrykofmaxim002** [comment]
Te quedò genial !

**Comment by Rild1998** [comment]
Te quedo genial

**Comment by Strato1257** [comment]
esta chida, voy atrasado pero espero poder ponerme al corriente!

**Comment by dzarkV** [comment]
Oye, está muy chévere!

**Comment by floressdi** [comment]
> Oye hermano donde aprendiste Tailwind css?. te quedo muyyyy bien. saludos

Basicamente Tailwind es como ya tener tus clases hechas en css solo las mandas a llamar pero ya esta hecho eso ahorra tiempo te paso la documentacion  saludos.
https://tailwindcss.com/
